Pod/Extensions/Forgot Password/SKYContainer+ForgotPassword.m

```diff
@@ -0,0 +1,41 @@
+//
+//  SKYContainer+ForgotPassword.m
+//  SKYKit
+//
+//  Copyright 2017 Oursky Ltd.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#import "SKYContainer+ForgotPassword.h"
+
+@implementation SKYContainer (ForgotPassword)
+
+- (void)forgotPasswordWithEmail:(NSString *)emailAddress
+              completionHandler:(void (^)(NSDictionary *, NSError *))completionHandler
+{
+    [self callLambda:@"user:forgot-password"
+                arguments:@[ emailAddress ]
+        completionHandler:completionHandler];
+}
+
+- (void)resetPasswordWithUserID:(NSString *)userID
+                           code:(NSString *)code
+                       password:(NSString *)password
+              completionHandler:(void (^)(NSDictionary *, NSError *))completionHandler
+{
+    [self callLambda:@"user:reset-password"
```

Comment:
The `user:reset-password` lambda function argument is changing: https://github.com/SkygearIO/forgot_password/pull/16/files#diff-5d586952e502d553206198500bcdf043R176